pkg/agent/apiserver/apiserver.go

```diff
@@ -124,10 +125,10 @@ func newConfig(bindPort int, enableMetrics bool, kubeconfig string) (*genericapi
 	// Set the PairName but leave certificate directory blank to generate in-memory by default.
 	secureServing.ServerCert.CertDirectory = ""
 	secureServing.ServerCert.PairName = Name
-	secureServing.BindAddress = net.ParseIP("0.0.0.0")
+	secureServing.BindAddress = ip.UnspecifiedAddrMap[ipFamily]
 	secureServing.BindPort = bindPort
 
-	if err := secureServing.MaybeDefaultWithSelfSignedCerts("localhost", nil, []net.IP{net.ParseIP("127.0.0.1")}); err != nil {
+	if err := secureServing.MaybeDefaultWithSelfSignedCerts("localhost", nil, []net.IP{ip.LocalhostAddrMap[ip.V4Family], ip.LocalhostAddrMap[ip.V6Family]}); err != nil {
```

Comment:
There are some variables for well-known addresses, maybe no need to define another map and keys.
It could just be `[]net.IP{net.ParseIP("127.0.0.1"), net.IPv6loopback}`

Reply:
Updated.